pkgs/by-name/wh/whitesur-icon-theme/package.nix

```
@@ -32,13 +32,13 @@ lib.checkListOfEnum "${pname}: theme variants"
   stdenvNoCC.mkDerivation
   rec {
     inherit pname;
-    version = "2024-09-07";
+    version = "v2025-02-10";
 
     src = fetchFromGitHub {
       owner = "vinceliuice";
       repo = pname;
```

Probably a good time to fix this anti-pattern (#277994)

```suggestion
      repo = "WhiteSur-icon-theme";
```